portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,69 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 24) + 1;
+  const imgUrl = 'images/jfew-' + imageIndex + '.jpg';
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+}
+
+// modalbtn opens the first modal
+function Intromodal() {
+  let button = document.getElementById("Intro");
+  let modal = document.getElementById("myIntroModal");
+  modal.style.display = "block";
+}
+
+// modalbtn2 opens the second modal
+function Picturemodal() {
+  let button = document.getElementById("Pictures");
+  let modal = document.getElementById("myPictureModal");
+  modal.style.display = "block";
+}
+
+// modalbtn3 opens the third modal
+function Connectmodal() {
+  let button = document.getElementById("Connect");
+  let modal = document.getElementById("myConnectModal");
+  modal.style.display = "block";
+}
+
+// spanbtn closes the first modal
```

Comment:
ditto: I think your functions could be named something like

function closeIntroModal() {